sdk/batch/microsoft-azure-batch/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 7.1.0-beta.1 (Unreleased)
```

Comment:
This happened since the current version listed is this without a changelog entry.